packages/components/image/src/use-image.ts

```diff
@@ -111,6 +111,7 @@ export function useImage(originalProps: UseImageProps) {
   const isImgLoaded = imageStatus === "loaded" && !isLoadingProp;
   const isLoading = imageStatus === "loading" || isLoadingProp;
   const isZoomed = originalProps.isZoomed;
+  const isError = !isLoading && imageStatus === "failed" ? true : false;
```

Comment:
`!isLoading && imageStatus === "failed"` already gives `true` or `false`. No necessary to use ternary operator here.